recipe/meta.yaml

```
@@ -20,6 +21,8 @@ requirements:
   host:
     - python
     - pip
+    - wheel
+    - setuptools
     - nose
```

```suggestion
    - nose >=1.0
```

https://github.com/openvenues/pypostal/blob/bf3762124e0b08231aa8f3d0cfee835403093002/setup.py#L26

---

_\<sigh\>_ I didn't check that either! 😆